### DISCUSSION TITLE
ATCP Support

### DISCUSSION THREAD

**Discussion by mbsmith** [Blightmud/Blightmud#280] [Ideas]
As I came to find this project via a post from the maintainer regarding blightmud's use on IRE muds.  I thought it might be prudent to request, and possibly work on ATCP support.

www.ironrealms.com/rapture/manual/files/FeatATCP-txt.html for the specification details.

In another client I have this implemented in C.  Shouldn't be too tricky to port to blightmud.  However just doing it from spec shouldn't be that difficult either.

Anyway, just food for thought.

**Comment by LiquidityC** [comment]
This could probably be done in lua quite easily. However, ATCP predates GMCP and all the IRE muds I've played have GMCP now. Is ATCP still viable or could a user just stick to GMCP and still have access to the same information?

**Comment by rjm11** [comment]
ATCP support was dropped in IRE games before 2017 and replaced with GMCP, so GMCP support should be all that is needed at this point.
